ext/nnstreamer/tensor_filter/tensor_filter_mediapipe.cc

```diff
@@ -0,0 +1,358 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer Tensor_Filter, Mediapipe Module
+ * Copyright (C) 2020 HyoungJoo Ahn <hello.ahn@samsung.com>
+ */
+/**
+ * @file   tensor_filter_mediapipe.cc
+ * @date   06 May 2020
+ * @brief  Mediapipe module for tensor_filter gstreamer plugin
+ * @see    http://github.com/nnstreamer/nnstreamer
+ * @author HyoungJoo Ahn <hello.ahn@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @todo   If there is other input type (not video), or multiple inputs, current system cannot handle it.
+ *
+ * This is the per-NN-framework plugin (mediapipe) for tensor_filter.
+ */
+#include <iostream>
+#include <string>
+#include <stdexcept>
+
+#include <nnstreamer_cppplugin_api_filter.hh>
+#include <tensor_common.h>
+
+#include "mediapipe/framework/calculator_framework.h"
+#include "mediapipe/framework/formats/image_frame.h"
+#include "mediapipe/framework/formats/image_frame_opencv.h"
+#include "mediapipe/framework/port/commandlineflags.h"
+#include "mediapipe/framework/port/file_helpers.h"
+#include "mediapipe/framework/port/opencv_highgui_inc.h"
+#include "mediapipe/framework/port/opencv_imgproc_inc.h"
+#include "mediapipe/framework/port/opencv_video_inc.h"
+#include "mediapipe/framework/port/parse_text_proto.h"
+#include "mediapipe/framework/port/status.h"
```

Comment:
We'd better provide -I${mediapipe-header-path} to compilers and use <path> instead of "path" for #include ${mediapipe-headers}.

Reply:
you mean shorter paths? or something else?